src/components/StoreBook.tsx

```diff
@@ -43,6 +44,20 @@ const StoreBook: FC<IBookProps> = ({
     };
     getBookData();
   }, [id]);
+
+  const incorrectQuantityHandler = (value: string) => {
+    if (value.includes('-')) {
+      return warningToast('You have tried to add negative value!', 'negative-value-m-warning');
+    }
+    if (value.includes(',')) {
+      return warningToast('You have tried to add float value!', 'negative-value-dot-warning');
+    }
+    if (value.includes('.')) {
+      return warningToast('You have tried to add float value!', 'negative-value-coma-warning');
+    }
```

Comment:
These ifs can be merged into a single one, I think there's no need to separate the cases.
```suggestion
    if (value.includes(',') || value.includes('.')) {
      return warningToast('You have tried to add float value!', 'float-value-warning');
    }
```